app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -439,6 +454,32 @@ class ProjectInformationActivity : AppCompatActivity() {
         }, { showToast(getString(R.string.could_not_download_video), Toast.LENGTH_LONG) })
     }
 
+    /**
+     * moves a video from the delete Directory to the copy Directory
+     *
+     * @param fileUrl url of the file to be moved
+     * @param deleteDirectory directory from which it has to be moved
+     * @param copyDirectory directory in which it is moved to
+     * @return true if move was a success or there was no need to move, false else
+     */
+    private fun movingVideo(fileUrl: String, deleteDirectory: File, copyDirectory: File): Boolean{
```

Comment:
The return value seems to be unused in the code ?
If it is the case, the function could be simplified by removing the return value.
Or will there be some future use of it ?